app/models/conversation.rb

```diff
@@ -1,9 +1,16 @@
 class Conversation < ActiveRecord::Base
-  validates :user_id_1,
-            presence: true
-  validates :user_id_2,
-            presence: true
+  belongs_to :sender, :foreign_key => :sender_id, class_name: 'User'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.